lib/honeycomb/integrations/rack.rb

```diff
@@ -31,9 +31,9 @@ def call(env)
       hny = env["HTTP_X_HONEYCOMB_TRACE"]
       client.start_span(name: "http_request", serialized_trace: hny) do |span|
         add_field = lambda do |key, value|
-          next unless value && !value.empty?
-
-          span.add_field(key, value)
+          unless value.nil? || (value.respond_to?(:empty?) && value.empty?)
```

Comment:
Nice find, it would be great to have a test for this!

Reply:
The existing test covers both the regex match (we don't get any "user.<whatever>" values without that fix) and the `.respond_to?(:empty)` check (we get a `NoMethodError: undefined method 'empty?' for 2002-03-04 05:06:07 -0500:Time` error). The missing test related to my change is for `value == false` (though the Warden integration could also use tests for the logic around choosing which context to use, too...)

Reply:
Ah great! That makes total sense